static/app/components/modals/dashboardWidgetLibraryModal/tabsButtonBar.tsx

```diff
@@ -89,21 +67,48 @@ export function TabsButtonBar({
       >
         {t('Custom Widget')}
       </CustomButton>
+      <LibraryButton
+        barId={TAB.Library}
+        onClick={() => {
+          if (activeTab === TAB.Library) {
+            return;
+          }
+          trackAdvancedAnalyticsEvent('dashboards_views.widget_library.switch_tab', {
+            organization,
+            to: TAB.Library,
+          });
+          if (defined(onAddWidget)) {
+            openDashboardWidgetLibraryModal({
+              organization,
+              dashboard,
+              customWidget,
+              initialSelectedWidgets: selectedWidgets,
+              onAddWidget,
+            });
+          }
+        }}
+      >
+        <div>
```

Comment:
Why does it need wrapped in this div?

Reply:
left over from when i was trying to get the buttons align, removed!